specification/marketplacecatalog/data-plane/readme.md

```diff
@@ -73,4 +73,5 @@ These settings apply only when `--tag=package-2023-05-01-preview` is specified o
 ```yaml $(tag) == 'package-2023-05-01-preview'
 input-file:
   - Products/preview/2023-05-01-preview/products.json
+  - Search/preview/2023-01-01-preview/search.json
```

Comment:
Ideally, this Search should also be 2023-05-01-preview api-version. (basically, all of them in 2023-05-01-preview tag should be api-version 2023-05-01-preview)

Let us know why you are adding 2023-01-01-preview Search into 2023-05-01-preview tag.